modules/stanford_course_views/stanford_course_views.views_default.inc

```diff
@@ -185,8 +185,8 @@ function stanford_course_views_views_default_views() {
   $handler->display->display_options['empty']['area']['empty'] = TRUE;
   $handler->display->display_options['empty']['area']['content'] = '<p>Current Courses displays all courses offered in the current quarter.</p>
 <p>&nbsp;</p>
-[hook_views_pre_build:courses/search]';
-  $handler->display->display_options['empty']['area']['format'] = 'filtered_html';
+<a class="btn" href="/courses/search">Search courses</a>';
+  $handler->display->display_options['empty']['area']['format'] = 'content_editor_text_format';
```

Comment:
Almost all of our views rely on different text formats. I made changes to just run the hook on all views